app/models/giver.rb

```diff
@@ -1,3 +1,7 @@
 class Giver < ActiveRecord::Base
-  validates_presence_of :full_name, :message
+  include Tokenable
+
+  validates_presence_of :full_name, :message, :email
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.